spec/controllers/job_definitions_controller_spec.rb

```diff
@@ -38,6 +38,68 @@
 
       expect(assigns(:definition)).to be_new_record
     end
+
+    context 'with dup_from params' do
+      subject! { get :new, params: { dup_from: definition.id } }
+      it do
+        expect(response).to have_http_status(:ok)
+        expect(response).to render_template('new')
+
+        expect(assigns(:definition)).to be_new_record
+        expect(assigns(:definition).name).to eq definition.name
+        expect(assigns(:definition).admins).to eq definition.admins + [controller.current_user]
```

Comment:
[nis]
[definition.admins, controller.current_user] seems to be better.

Reply:
`expect(assigns(:definition).admins).to eq [definition.admins, controller.current_user]` is not working.

Reply:
oh..sorry, I misunderstood.  `definition.admins` is array.